### PR TITLE
Archana assignment 1

### PR DESCRIPTION
**Pull request by ArchanaNJun20**
-change in code

### CODE REVIEW

File: src/archanaNJun20/Example2.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+		String name;
+		
+		void DisplayData() {
+			System.out.println(rno + " " + name);
+		}
+		
+		public static void main(String[] args) {
+			Example1 example1 = new Example1(); // name="M" [I] // rno=10 [S]
+			example1.name = "Maulik";
+
```
Comment (KrishnaKTechnocredits) [inline src/archanaNJun20/Example2.java]: Remove space.

File: src/archanaNJun20/Example2.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package archanaNJun20;
+
+public class Example2 {
+	
+		static int rno;
+		String name;
+		
+		void DisplayData() {
```
Comment (KrishnaKTechnocredits) [inline src/archanaNJun20/Example2.java]: Method name in small letter.

File: src/archanaNJun20/Example2.java (hunk 1 above)
Comment (ArchanaNJun20) [reply src/archanaNJun20/Example2.java]: Done

File: src/archanaNJun20/Example2.java (hunk 2 above)
Comment (ArchanaNJun20) [reply src/archanaNJun20/Example2.java]: Done

**Pull request closed without merging**